integration-tests/src/test/java/org/apache/fineract/integrationtests/AdvancedPaymentAllocationLoanRepaymentScheduleTest.java

```diff
@@ -2962,6 +2962,32 @@ public void uc115() {
         }
     }
 
+    // UC116: Advanced payment allocation, Cumulative, horizontal repayment processing, mixed grouping of allocation rules
```

Comment:
Please change the description accordingly what the test is testing

Reply:
Done!